src/mixins/filing-template-mixin.ts

```diff
@@ -153,6 +154,7 @@ export default class FilingTemplateMixin extends DateMixin {
         correctedFilingType: this.getCorrectedFilingType,
         correctedFilingDate: this.getCorrectedFilingDate,
         comment: `${this.defaultCorrectionDetailComment}\n${this.getDetailComment}`,
+        ...(this.hasOnlyDetailDataChanged ? { commentOnly: true } : {}),
```

Comment:
Did you get an answer that this applies to all business types (corps, coops, firms)?

Reply:
![image](https://github.com/user-attachments/assets/551232ae-96c2-4f86-8609-575185bfd3ca)